src/css/start-script-generator.css

```diff
@@ -0,0 +1,198 @@
+.server-config-container {
+  background-color: #222;
```

Comment:
I know nothing about the docs, but I would expect that we have some variables for color so that they respect the dark/light theme switch

Reply:
Yep. Forgot people use light theme. Will address this